packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1126,9 +1126,17 @@ class RouteObserver<T extends Route<dynamic>> extends NavigatorObserver {
   void subscribe(RouteAware routeAware, T route) {
     assert(routeAware != null);
     assert(route != null);
-    if (!_listeners.containsKey(route)) {
+
+    Set<RouteAware> subscribers = _listeners[route];
+
+    if (subscribers == null) {
+      subscribers = new Set<RouteAware>();
+
+      _listeners[route] = subscribers;
+    }
+
```

Comment:
this code is simple enough that you can skip the blank lines here, it'll still be clear.

Reply:
(same with the methods below. Avoid blank lines in otherwise short methods.)